File: JavaGuiHttpReq/LekerApi/src/Kolcsonzesek.java
public class Kolcsonzesek {
    Integer kolcsonzes_id;
	Integer kolcsonzo_id ;
	String iro;
	String mufaj;
    String cim;

    public Kolcsonzesek(
        Integer kolcsonzes_id,
        Integer kolcsonzo_id ,
        String iro,
        String mufaj,
        String cim
    ) {
        this.kolcsonzes_id = kolcsonzes_id;
        this.kolcsonzo_id  =kolcsonzo_id;
        this.iro =iro;
        this.mufaj =mufaj;
        this.cim =cim;
    }
}

File: api/konyvtar_api/app/Models/Kolcsonzesek.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use App\Models\Kolcsonzok;

class Kolcsonzesek extends Model
{
    use HasFactory;

    protected $table = "kolcsonzesek";
    protected $primaryKey = "Kolcsonzes_id";

    public $timestamps = false;

    protected $fillable =[
        "kolcsonzo_id",
        "iro",
        "mufaj",
        "cim"
    ];

    public function kolcsonzok(){                                   //harmadiki primary key a fö táblában ami hez belogsTo ez a tábla
        return $this->belongsTo(Kolcsonzok::class, "kolcsonzo_id", "kolcsonzo_id");
    }                                            //második paraméter a FK(a kulcs ami a fö táblára vonatkozik ebben a táblában)

}
